frontend/test/__support__/e2e/cypress.js

```diff
@@ -33,7 +33,7 @@ export function main() {
 }
 
 export function sidebar() {
-  return cy.get(".scroll-y");
+  return cy.findByTestId("sidebar");
```

Comment:
Learning we do not accessibility tagging here.

Perhaps we could start.

So if we will add attrs that a user browser will see for testing, we may prefer to use aria `role`s for example instead of `data-testid`.

Then we can do `cy.findByRole(…)`.

Reply:
Some useful links:
- https://www.digitala11y.com/wai-aria-1-1-cheat-sheet/
- https://a11y-style-guide.com/style-guide/
- https://accessibility.psu.edu/

Reply:
I used `aside` for sidebars instead. However, with `sidebar-right` or `sidebar-left` I think it is better to keep test ids because they might be rendered at the same time and it is better than rely on indices